app/src/main/res/values/strings.xml

```diff
@@ -2,6 +2,7 @@
     <string name="app_name">open-event-android</string>
     <string name="Events">Events</string>
     <string name="Profile">Profile</string>
+    <string name="Likes">Likes</string>
```

Comment:
`likes`

And technically should be favorites

Reply:
yes it should be favorites but we are trying to keep it consistent with EventBrite and they call it likes? Should i change this to favourites still?

Reply:
OK, then no

Also does eventbrite show favorites as a separate bottom navigation panel

Reply:
yes

Reply:
resolved conflicts